packages/documentation/docs/documentation/useHotkeys/basic-usage.mdx

```diff
@@ -149,6 +149,47 @@ Since we pressed down one hotkey already the example above started at the count
 globally if we don't use its return value.
 :::
 
+***
+
+## Sequential Hotkeys
+
+We can also listen to keystrokes in a sequential way. This means that the user has to press the keys in the order we specify. To specify a sequential hotkey, we use the `>` character.
+
+```jsx live
+function ExampleComponent() {
+  const [count, setCount] = useState(0)
+  useHotkeys('g>h>i', () => setCount(prevCount => prevCount + 1))
+
+  return (
+    <span>Received the combination {count} times.</span>
+  )
+}
+```
+
+In the example above, the user has to press the `g` key, then the `h` key and then the `i` key in order to trigger the callback.
+
+#### Timeout
+The default timeout for the sequential hotkey is `1000ms`. If the user doesn't press the next key in time, the sequence is aborted.
```

Comment:
```suggestion
`useHotykes` sets a default time window for the sequential hotkey of `1000ms` to prevent listening for incomplete sequences indefinitely. If the user doesn't press the next key in that given time, the sequence is aborted.
```